### PR TITLE
Add implementation for ClassesUi in DG

### PR DESCRIPTION
**Pull request by yourally2**
Adds a sequence diagram and how it works

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -233,6 +233,24 @@ implement `Predicate<Student`, and predicate class for filtering `TuitionClass`
 Each custom predicate class contains a `List` of search strings that would be used to match against the tested items 
 in the search.
 
+### Class Panel feature
```
Comment (s7u4rt99) [inline docs/DeveloperGuide.md:236]: Do you want to name it `Classes` feature instead? To match the name of our tabs

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -233,6 +233,24 @@ implement `Predicate<Student`, and predicate class for filtering `TuitionClass`
...
+The class panel feature allows one to see the user's classes and each class' corresponding students. 
+
+#### Implementation
+![Structure of ClassPanel](images/ClassPanelUiClassDiagram.png)  
+The class diagram for Class Panel as shown in the [ClassesUI component](#classpanel-ui) is replicated here for convenience.
```
Comment (s7u4rt99) [inline docs/DeveloperGuide.md:241]: This line as well

**Review by s7u4rt99 — COMMENTED** [review]
Other than this, LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (yourally2) [reply docs/DeveloperGuide.md:236]: Ok I'll change the name to Classes Ui feature, Classes feature sounds a bit too vague

**Comment by s7u4rt99** [comment]
LGTM

**Pull request merged**